bndtools.jareditor/resources/plugin.xml

```diff
@@ -14,4 +14,55 @@
 			<run class="bndtools.jareditor.internal.JarFileSystem" />
 		</filesystem>
 	</extension>
+	<extension point="org.eclipse.ui.navigator.navigatorContent">
+		<commonFilter
```

Comment:
Does this hiding of the temp project also work with the new `Bndtools Explorer` view?

Reply:
Yes: That was added here: https://github.com/bndtools/bnd/pull/3711/files#diff-05adbc15428a40a3be593eb756feb105R51-R58